custom/opm/health_status.py

```diff
@@ -320,3 +356,40 @@ def adult_scale(self):
     @property
     def child_scale(self):
         return self.service_available('vhnd_child_scale_available')
+
+    @property
+    def ifa_stock_available(self):
```

Comment:
didn't test the vhnd specific columns locally, they all give zero sine the vhnd table doesn't have any rows  (had some problem with running VhndFluff). Can you test these on your local machine? If not, I will try it tomorrow/